test/e2e/apimachinery/crd_publish_openapi.go

```diff
@@ -49,7 +49,7 @@ var (
 	metaPattern              = `"kind":"%s","apiVersion":"%s/%s","metadata":{"name":"%s"}`
 )
 
-var _ = SIGDescribe("CustomResourcePublishOpenAPI [Feature:CustomResourcePublishOpenAPI]", func() {
+var _ = SIGDescribe("CustomResourcePublishOpenAPI", func() {
```

Comment:
Are there any places where the gate is set via the test framework?

Reply:
yes, previously alpha [pull](https://github.com/kubernetes/test-infra/pull/11618/files) and [CI](https://github.com/kubernetes/test-infra/pull/12070/files) jobs looked for regex and enabled `api/all=true`. With this change now the test runs in non-alpha jobs (e.g. [blocking presubmit](https://github.com/kubernetes/test-infra/blob/339ad5a261b51893e26fb62213b9de9d2a9dd97f/config/jobs/kubernetes/sig-gcp/sig-gcp-gce-config.yaml#L21-L58)) which don't modify gates. I'll open a pull to remove the regex pattern from alpha jobs